src/app.ts

```diff
@@ -44,27 +60,68 @@ export class UmbAppElement extends UmbLitElement {
 		},
 	];
 
+	#authFlow: UmbAuthFlow;
 	#umbIconRegistry = new UmbIconStore();
 	#uuiIconRegistry = new UUIIconRegistryEssential();
 	#runtimeLevel = RuntimeLevelModel.UNKNOWN;
+	#isMocking = import.meta.env.VITE_UMBRACO_USE_MSW === 'on';
```

Comment:
Just a small note for ourselves. I think we should look into moving all mocking logic outside of the app element, so the element doens't know anything about what is serving the data.

The code is already like this, so it is fine to look into this outside of this PR.

Reply:
I have added a TODO